pkgs/development/python-modules/types-pyopenssl/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, cryptography
+}:
+
+buildPythonPackage rec {
+  pname = "types-pyOpenSSL";
```

`pname` should be normalized according to PEP-503, see also https://nixos.org/manual/nixpkgs/stable/#contributing

```suggestion
  pname = "types-pyopenssl";
```

---

it doesn't find the package on pypi when normalized